distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column_Vector_Extensions.enso

```diff
@@ -0,0 +1,53 @@
+from Standard.Base import all
+
+import project.Data.Column.Column
+
+## Creates a column from a `Vector` of values.
+
+   Arguments:
+   - name: Name of the new column.
+Vector.to_column : Text -> Column
+Vector.to_column self name="Vector" =
+    Column.from_vector name self
+
+## Creates a column from a `Range` of values.
+
+   Arguments:
+   - name: Name of the new column.
+Range.to_column : Text -> Column
+Range.to_column self name="Range" =
+    Column.from_vector name self.to_vector
+
+## Creates a column from a `Date_Range` of values.
+
+   Arguments:
+   - name: Name of the new column.
+Date_Range.to_column : Text -> Column
+Date_Range.to_column self name="Date_Range" =
+    Column.from_vector name self.to_vector
+
+## Compute a single statistic on the column.
+
+   Arguments:
+   - statistic: Statistic to calculate.
+Column.compute : Statistic -> Any
+Column.compute self statistic=Statistic.Count =
+    self.compute_bulk [statistic] . first
+
+## Compute statistics on the column.
+
+   Arguments:
+   - statistics: Set of statistics to calculate.
+Column.compute_bulk : Vector Statistic -> Vector Any
+Column.compute_bulk self statistics=[Statistic.Count, Statistic.Sum] =
+    Statistic.compute_bulk self.to_vector statistics
```

Comment:
I'm wondering if this shouldn't return a single row table?

Well, I guess there are pros and cons:

Pros:
+ better for visualization - we could ensure the header includes the name of each statistic so it is clear which is which

Cons:
- a bit harder to get the nth value from it (instead of `r.at i` we need `r.first_row.at i`)
- not consistent with `compute_bulk` on Vector (but IMO that is OK because it is more consistent with how most Table/Column operations are Table-oriented)

I guess for now it is good to keep it as is. I'm wondering if we could do some magic annotation to the returned Vector to visualize it with the statistics named somehow - but that seems possibly a bit too magical.

Reply:
I'll make it a Table.